torchdata/datapipes/iter/load/huggingface.py

```diff
@@ -17,42 +17,48 @@
 
 
 def _get_response_from_huggingface_hub(
-    dataset: str, split: str, revision: str, streaming: bool, data_files: Optional[Dict[str, str]]
+    path: str, streaming=True, split="train", revision="main", **config_kwargs
```

Comment:
nit: Could you please add type annotation to each argument?

Reply:
nit: can we preserve the previous ordering of arguments for the most backward compatibility? (unless there is a reason why it should be changed)